django_perf_rec/orm.py

```diff
@@ -21,11 +23,12 @@ def patch_ORM_to_be_deterministic():
     if patch_ORM_to_be_deterministic.have_patched:
         return
     patch_ORM_to_be_deterministic.have_patched = True
-
     patch_QuerySet()
     patch_Query()
     patch_Q()
-    patch_delete()
+    version = django.get_version()
+    if version.startswith('1.8') or version.startswith('1.9'):
```

Comment:
> moumoutte@gloupf:~/projects/django-perf-rec$ .tox/py27-django18/bin/python
Python 2.7.13 (default, Jan 19 2017, 14:48:08) 
[GCC 6.3.0 20170118] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import django
>>> django.VERSION
```
(1, 8, 17, 'final', 0)
>>> django.VERSION <= (1,8)
False
```